queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
+SELECT date_of_birth FROM animals WHERE date_of_birth BETWEEN '2016/01/01' AND '2019/12/31';
+SELECT escape_attempts FROM animals WHERE neutered = false AND escape_attempts < 3;
```

Comment:
- [ ] This query returns the `escape_attempts` of all animals that are not neutered and have less than 3 escape attempts. However, we are asked to list the name of all animals that are neutered and have less than 3 escape attempts. A good query would be `SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;`